dev_docs/getting_started/hello_world_plugin.mdx

```diff
@@ -127,13 +127,19 @@ export function plugin() {
 
 ## 2. Option 2 - Use the automatic plugin generator
 
+---
```

Comment:
@mistic I wonder if we should remove the whole "option 2" until https://github.com/elastic/kibana/issues/162065 is completed?